geant4-data-rpm.file

```diff
@@ -10,9 +10,11 @@
 
 %if "%{?source0:set}" != "set"
 %define source0  http://geant4.cern.ch/support/source/%{G4DataTool}.%{realversion}.tar.gz
+%define source1 http://cmsrep.cern.ch/cmssw/download/geant4/%{G4DataTool}.%{realversion}.tar.gz
```

Comment:
@mrodozov , no you should not change this. You should add 
```
%define source1 http://cmsrep.cern.ch/cmssw/download/geant4/<TOOLNAMe>.%{realversion}.tar.gz
```
in each of the `geant4-<TOOL>.spec` for which we have to get data from this special location